### PR TITLE
Expression problem procedural f#

### PR DESCRIPTION
**Pull request by Caspersn2**

### CODE REVIEW

File: benchmarks/expression_problem/procedural_f#/Program.fs [hunk 1]
```diff
@@ -0,0 +1,104 @@
+﻿// Learn more about F# at http://fsharp.org
+
+open System
+open System.Text
+
+type IExpr = 
+    abstract Print: string
```
Comment (Nitram1995) [inline benchmarks/expression_problem/procedural_f#/Program.fs:7]: Abstract? I procedural?

File: benchmarks/expression_problem/procedural_f#/Program.fs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+type ADD =
+    {
+        Left: IExpr
+        Right: IExpr
+    }
+    member this.Print = 
+         "(" + this.Left.Print + "+" + this.Right.Print + ")"
+    member this.Eval =
+        this.Left.Eval + this.Right.Eval
```
Comment (Nitram1995) [inline benchmarks/expression_problem/procedural_f#/Program.fs:18]: Member functioner? I procedural

File: benchmarks/expression_problem/procedural_f#/Program.fs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    interface IExpr with
```
Comment (Nitram1995) [inline benchmarks/expression_problem/procedural_f#/Program.fs:19]: nahhh

**Review by Nitram1995 — CHANGES_REQUESTED** [review]
Vil mene, at det her er OOP

File: benchmarks/expression_problem/procedural_f#/Program.fs (hunk 3 above)
Comment (Caspersn2) [reply benchmarks/expression_problem/procedural_f#/Program.fs:19]: nahhh?? Kan du ikke lide mit 100% procedural program?

**Pull request closed without merging**